stl/inc/format

```diff
@@ -2864,7 +2864,7 @@ _NODISCARD _OutputIt _Fmt_write(
 
     auto _To_upper  = false;
     auto _Format    = chars_format::general;
-    auto _Exponent  = '\0';
+    auto _Exponent  = 'e';

```

Comment:
Suggestion: This makes the following assignments `_Exponent = 'e';` dead; I think we should remove them. The resulting code will be easier to understand: usually we print decimal, where `'e'` is the proper exponent, but for hexfloats we need `'p'` to be the exponent. (The decimal exponent can actually appear for scientific and general; it can't appear for fixed so initializing it to `'e'` is unnecessary but harmless.)